src/__tests__/__snapshots__/SearchResults.test.js.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SearchResults renders correctly 1`] = `
+<DocumentFragment>
+  <div
+    class="search-results-container"
+  >
+    <img
+      class="card-image"
```

Comment:
Look at your actual components output in the browser and youll see how much is missing from this, left a note on the test file about how to fix this